Dixit_Logic/Classes/DixitGame.cs

```diff
@@ -49,10 +49,15 @@ public IGameState ActualGameState
         }
 
         /// <summary>     
-        /// This event is triggered at the start of a trun when all cards are consumed from the MainDeck and players have no more cards too .
+        /// This event is triggered when all cards are consumed from the MainDeck and players have no more cards too .
```

Comment:
If this comment is true, then the implementation is wrong. A Dixit game lasts until one of the players reach 30 points. Playing until all cards are gone is not a reliable metric as the number of cards can be vast.